### DISCUSSION TITLE
Selecting a logo for mycoriza

### DISCUSSION THREAD

**Discussion by tiranuom** [hsenid-mobile/mycoriza#3] [Show and tell]
**Mycoriza** as name suggests, is the symbiosis relationship between the mycelium network and the root. In this case network and the react application. With that in mind, I have come up with 2 similar logos. I need the confirmation / ideas on this. The logos are attached herewith.

1. 
<img alt="logo" height="150" src="https://user-images.githubusercontent.com/853253/133914583-7b94c95d-9c90-41ac-a301-f366052df872.png" width="150"/>

2.
<img alt="logo2" height="150" src="https://user-images.githubusercontent.com/853253/133914586-b8bfb90e-a077-427b-b652-ef5d49051929.png" width="150"/>

Please provide the feedback on it

**Comment by sandarenu** [comment]
I like the option 2

**Comment by LahiruJayasekara** [comment]
+1 for option 2

On Mon, Sep 20, 2021 at 5:51 PM Chathurika Sandarenu <
***@***.***> wrote:

> I like the option 2
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/hsenid-mobile/mycoriza/discussions/3#discussioncomment-1356935>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGLN7TD4EVDN7TLVCPTGH7TUC4ROBANCNFSM5EJ4AXVQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Lahiru Jayasekara
0716492170

**Comment by LahiruJayasekara** [comment]
+1 for option 2

**Comment by tiranuom** [comment]
I added the `option 2` in the documentation
